### PR TITLE
[LPT] issue #37332: tests unification

### PR DESCRIPTION
**Pull request by v-Golubev**
Validation:
* accuracy: developer-services/job/accuracy/829/
* performance: developer-services/job/performance/472/

Validation:
* accuracy: developer-services/job/accuracy/791/  (reference: developer-services/job/accuracy/800/)
* performance: developer-services/job/performance/466/

### CODE REVIEW

**Comment by eshoguli** [comment]
remote `[LPT] removeStandaloneCleanup + GPU transformations update` commit from PR, please

**Comment by v-Golubev** [comment]
> remove `[LPT] removeStandaloneCleanup + GPU transformations update` commit from PR, please

This commit was removed

**Comment by eshoguli** [comment]
everything is OK for me

File: inference-engine/tests/functional/plugin/shared/src/low_precision_transformations/concat_with_intermediate_transformation.cpp [hunk 1]
```diff
@@ -72,6 +72,35 @@ void ConcatWithIntermediateTransformation::SetUp() {
         transparentIntermediate,
         { 256ul, ngraph::Shape({}), {0.f}, {2.55f}, {0.f}, {2.55f} },
         { 256ul, ngraph::Shape({}), {0.f}, {2.55f}, {0.f}, {2.55f / 2.f} });
+
+    validate();
+}
+
+void ConcatWithIntermediateTransformation::validate() {
```
[the remark this answers is not in the document]
Comment (eshoguli) [reply inference-engine/tests/functional/plugin/shared/src/low_precision_transformations/concat_with_intermediate_transformation.cpp:79]: Validate method implementation is unique for each test and depends on used model. Let's keep it as is.

**Review by iefode — APPROVED** [review]

**Review by dmitry-gorokhov — APPROVED** [review]

**Pull request merged**